javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinter.java

```diff
@@ -65,6 +67,7 @@
 
     /**
      * Parse the code and setup the LexicalPreservingPrinter.
+     * @deprecated just use the other constructor.
```

Comment:
Uhuh, we might even refactor this code into the node constructors ("I know my start token, my end token, and the start and end token of my children, so I can deduce which tokens are mine")